src/main/java/edu/iis/powp/commandtransformer/decorator/gui/events/ApplyFlipCommandListener.java

```diff
@@ -0,0 +1,33 @@
+package edu.iis.powp.commandtransformer.decorator.gui.events;
+
+import edu.iis.powp.command.IPlotterCommand;
+import edu.iis.powp.command.manager.PlotterCommandManager;
+import edu.iis.powp.commandtransformer.decorator.FlipCommandDecorator;
+import edu.iis.powp.features.CommandsFeature;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class ApplyFlipCommandListener implements ActionListener {
+
+    private JCheckBox flipYCheckBox;
```

Comment:
"Direct dependency to GUI elements. The state of the GUI could be accessed through dedicated interface." to be done